diesel_tests/tests/types_roundtrip.rs

```diff
@@ -4,7 +4,7 @@ pub use quickcheck::quickcheck;
 use self::chrono::{Duration, NaiveDate, NaiveDateTime, NaiveTime};
 use self::chrono::naive::date;
 
-pub use schema::{connection, TestConnection};
```

Comment:
I'm not sure about this change or the next one, but I couldn't get it to compile on my nightly without it.

```
% rustc --version
rustc 1.9.0-nightly (c9629d61c 2016-03-10)
```